source/Irrlicht/COpenGLDriver.h

```diff
@@ -154,7 +154,12 @@ namespace video
         } vertexInputParams;
 
         struct {
-            core::smart_refctd_ptr<const COpenGLDescriptorSet> descSets[video::IGPUPipelineLayout::DESCRIPTOR_SET_COUNT];
+            struct SDescSetBnd {
+                core::smart_refctd_ptr<const COpenGLPipelineLayout> pplnLayout;
+                core::smart_refctd_ptr<const COpenGLDescriptorSet> set;
+                core::smart_refctd_dynamic_array<uint32_t> dynamicOffsets;
+            };
+            SDescSetBnd descSets[IGPUPipelineLayout::DESCRIPTOR_SET_COUNT];

```

Comment:
I'd actually move that up to CNull driver.

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/358#discussion_r334517945